Sources/PureSwiftUI/Extensions/Convenience/CoreGraphics/CGRect+Convenience.swift

```diff
@@ -183,9 +183,13 @@ public extension CGRect {
 // MARK: ----- INSET
 
 public extension CGRect {
-    
+
     func inset(_ top: CGFloat, _ leading: CGFloat, _ bottom: CGFloat, _ trailing: CGFloat) -> CGRect {
-        inset(by: UIEdgeInsets(top: top, left: leading, bottom: bottom, right: trailing))
+      #if os(macOS)
+      return self
+      #else
+      return inset(by: UIEdgeInsets(top: top, left: leading, bottom: bottom, right: trailing))
+      #endif
     }
```

Comment:
I wasn’t sure what to replace this with on macOS, would be great if you could take a look!